### PR TITLE
Add/activitylog

### PR DESCRIPTION
**Pull request by RachelleT**
Implementing UI tests to verify the activity log within the app by 

1) Publishing a post
2) Checking for the post in the activity log

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -3,14 +3,12 @@
...
-        <entry key="active-configuration" value="LOCAL_FILE:$PROJECT_DIR$/config/checkstyle.xml:a8c Style" />
-        <entry key="checkstyle-version" value="8.2" />
+        <entry key="checkstyle-version" value="8.20" />
         <entry key="copy-libs" value="false" />
         <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
-        <entry key="location-2" value="LOCAL_FILE:$PROJECT_DIR$/config/checkstyle.xml:a8c Style" />
         <entry key="scan-before-checkin" value="false" />
-        <entry key="scanscope" value="AllSourcesWithTests" />
+        <entry key="scanscope" value="JavaOnly" />
```
Comment (JavonDavis) [inline .idea/checkstyle-idea.xml]: Why are you making changes to the `.idea` files? This was probably done automatically by your IDE, you'll need to undo this.

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/EditorTests.java [hunk 2]
```diff
@@ -105,4 +105,25 @@ public void testPublishFullPost() {
         boolean isPublished = editorPage.publishPost();
         assertTrue(isPublished);
     }
+
+    @Test
+    public void testPublishSimplePage() {
```
Comment (JavonDavis) [inline WordPress/src/androidTest/java/org/wordpress/android/e2e/EditorTests.java]: This test involves publishing a simple page? Was that intended to be a part of this PR, from what I understood this PR was only to contain the test for the activity log?

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java [hunk 3]
```diff
@@ -4,12 +4,20 @@
...
+import androidx.test.espresso.matcher.ViewMatchers;
 
 import org.wordpress.android.R;
 
 import static androidx.test.espresso.Espresso.onView;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.hasSibling;
 import static androidx.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA;
```
Comment (JavonDavis) [inline WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java]: Seems to be a lot of unused imports added here, can we remove any we're not using please.

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/EditorPage.java [hunk 4]
```diff
@@ -106,4 +106,11 @@ public boolean publishPost() {
         waitForElementToBeDisplayed(publishConfirmation);
         return isElementDisplayed(publishConfirmation);
     }
+
+    public boolean publishPage() {
```
Comment (JavonDavis) [inline WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/EditorPage.java]: This function seems to be unrelated to the activity log test as you described, if the PR wasn't intended to cover the page publish I'd recommend removing that code from this PR.

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/components/MasterbarComponent.java [hunk 5]
```diff
@@ -17,4 +17,15 @@ public MasterbarComponent clickBlogPosts() {
         clickOn(R.id.row_blog_posts);
         return this;
     }
+
+    public MasterbarComponent clickSitePages() {
```
Comment (JavonDavis) [inline WordPress/src/androidTest/java/org/wordpress/android/e2e/components/MasterbarComponent.java]: Same comment as before about code unrelated to the activity log test.

File: WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/SitePickerPage.java [hunk 6]
```diff
@@ -1,14 +1,27 @@
 package org.wordpress.android.e2e.pages;
 
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.espresso.action.ViewActions;
+
+import org.wordpress.android.R;
+
 import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
```
Comment (JavonDavis) [inline WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/SitePickerPage.java]: Seem to add a lot of new imports that aren't used, best we remove them.

**Comment by oguzkocer** [comment]
Similar to my comment in #10454, this PR contains a lot of commits that it shouldn't have. @RachelleT could you please checkout a new branch from current `develop` and apply your changes there and open a fresh PR? Please make sure that it doesn't contain any changes to the `.circleci/config.yml` file and it doesn't have any changes that are not essential to the PR. Also, if you can add a clear description for the goal of the PR, that'd be great.

Some of the extra files don't seem necessary at a first glance, but I might be missing how they are used, so once you open the PR, we can look into that as well.

If you have any questions about any of that, please don't hesitate to ping me. Thanks!
